scss/_modal.scss

```diff
@@ -189,18 +198,19 @@
   display: flex;
   flex-shrink: 0;
   flex-wrap: wrap;
+  gap: var(--#{$prefix}modal-footer-gap);
```

Comment:
Do you have more info about why this modification was postponed to v6 in Bootstrap? (see `// Todo in v6: replace with gap on parent class`)

Reply:
I made this modification before the bootstrap commit arrived, so I didn't knew about that. It was in https://github.com/twbs/bootstrap/pull/36060 and I can't see any info about this.